### DISCUSSION TITLE
Add option to use arrow keys to go between attachment previews

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5747] [Feature Requests]
This would be the same on a record or in the Attachments viewer page

https://user-images.githubusercontent.com/37256050/221669063-d8cffb04-378e-48c1-af35-0d861c7abaad.mov

Navigation could be performed by pressing ⬅ or ➡️ keys rather than the buttons (since they shift place)

**Comment by maxpatiiuk** [comment]
This can be included as part of https://github.com/specify/specify7/issues/1746

**Comment by grantfitzsimmons** [comment]
Will be solved by #3632 once it is merged.
